packages/backend/src/peripherals/aztec/schemas.ts

```diff
@@ -28,3 +31,32 @@ export const GetRollupResponseBodySchema = z.object({
     rollup: Rollup,
   }),
 })
+
+export function parseWithSchema<T extends z.ZodType<unknown>>(
```

Comment:
Re-thrown error includes also the original `data` - zod only tells us what was different than expected, does not provide the whole input. In this case where we don't really know what can come from the api we'd rather see the whole body, because it gives more context for debugging.

Reply:
Okay. Let's keep it for now then